editor/components/post-schedule/index.js

```diff
@@ -25,6 +26,8 @@ export function PostSchedule( { date, onUpdateDate } ) {
 		<DateTimePicker
 			key="date-time-picker"
 			currentDate={ date }
+			utcOffset={ settings.timezone.offset }
+			todayButton={ __( 'Today' ) }
```

Comment:
Do we want this in all DatePicker components? in which case it's probably better to add it to the DatePicker component instead.

Also, why do we need the `utcOffset` here?

Reply:
I don't think it needs to be in all the date pickers. Some might use to to select date of birth or other uses, and won't be needed there. 

We need ofset to make sure the "Today" it chooses is according to the timezone set in WordPress settings.

I'm in India and my local setup has GMT 0 timezone (23rd of May), so when I clicked on Today around 1am IST time (24th of May), and WordPress took it as a future"Scheduled" post.

Reply:
I realize my explanation was really confusing. Basically to make sure "Today" is taken from the timezone defined in WP settings, not the browser...which will cause the posts to be published in past or future if you're not from the same timezone as your WP. Hope this makes more sense. 😁

Reply:
Thanks for the clarification, it makes sense but do you think we should avoid using the React Date Picker props and think of better prop names/values because the underlying implementation can change later.

I think `utcOffset` make sense but instead of `todayButton` I'd prefer a boolean only prop. `showTodayButton`. And let's document these props in the README of the `DateTimePicker`  component

Reply:
I was thinking of doing the same thing but seemed like an overkill to simply things. But didn’t think of the possibility of react-date-picker ever changing their implementation. I’ll update the PR tonight. :)

Reply:
Actually, I was thinking of us moving away from react-date-picker ;) #1311

Reply:
It says it is accessible but doesn't look like it at first glance at all. And you've confirmed that it's not that assessible. XD

I think the best way will be to fork it and make it accessible.

Reply:
Wanted to ask you the best way to implement the solution you proposed in JSX. I've added `showTodayButton` to `<DateTimePicker>` element, and checked the condition in `ReactDatePicker` with the following:

```
todayButton={ showTodayButton ? __( 'Today' ) : undefined }
utcOffset={ showTodayButton ? settings.timezone.offset : undefined }
```

Or should I do it using a spreaded object instead?

Reply:
i'm fine with that :) but I think instead of accessing the date settings in the component I'd just pass an `utcOffset` prop like you did originally to avoid having to add a dependency to the `date` module in the `components` module.

Reply:
Let me know if these changes are good. :)